Recommendations Block: Update documentation (#821)
Parsely/wp-parsely: src/blocks/recommendations/README.md

# Parse.ly Recommendations Block

> The Recommendations Block is designed to showcase links to content on your site as provided by the [Parse.ly `/related` API endpoint](https://www.parse.ly/help/api/recommendations#get-related). You can add it in Posts, Pages, or nearly any other custom post type. The Block can also be used in Full Site Editing (FSE) mode or as a [Block-based Widget](https://wordpress.org/support/article/block-based-widgets-editor/).

Reviewer: Should we mention that this replaces the legacy widget?

Author: We could argue that the Block replaces the Widget, but currently for this to be true the following must apply:
- Running WP 5.9 or later.
- Not having disabled Block widgets through a plugin like Classic Widgets.

So the Block is a replacement, but not in all cases currently. We could add a mention like:
> The Block is targeted to completely replace the `Recommended Widget` which will be removed in wp-parsely version 4.0.0. If you're using the Widget, we highly recommend to consider the Block and start using it instead.

Author: I've added a note about this.

## Requirements

> The Block is available only in WordPress 5.9 and higher.

Reviewer: ```suggestion
The Block is available only in WordPress 5.9 or later.
```

### Maximum Results

Specify the maximum number of recommendations to show.

> Note: In certain cases, the number of results might be lower than this setting (especially if it is set to a large number).

### Show Images

Toggle this setting to enable or disable images in the results. 

### Image Style

Use this setting to specify whether the Block should display the post's original image or the respective thumbnail provided by Parse.ly. Selecting the thumbnail option could be more performant but is not ideal for displaying large images.

> Note: If you select the `Parse.ly thumbnail` setting, the image will appear big and distorted. You should use CSS to tailor it to your needs.

### Sort Recommendations

Sort the results by score or published date. 

### Boost

Sub-sort the results by a variety of [available metrics](https://www.parse.ly/help/api/available-metrics).

### Additional CSS class(es) (Advanced section)

For styling purposes, you can specify one or more CSS classes that will be assigned to the block. 

## FAQ

### I can't find the Block.

> Ensure that you're running WordPress version 5.9.0 or higher, as the Block is unavailable in lower WordPress versions.

Reviewer: Make sure they don't have the Proxy disabled?

Author: Good catch. I had thought about it but then forgot.

Author: When the proxy is disabled, the block itself will display: `The REST route is unavailable. To use it, wp_parsely_enable_related_api_proxy should be true.`.

I think that this should be enough, but let me know if you would also like to have it in the docs.

Reviewer: I agree it's enough, no need to include it in the docs.
